packages/commonwealth/client/scripts/views/components/Header/UserDropdown/useCheckAuthenticatedAddresses.ts

```diff
@@ -26,7 +26,7 @@ const useCheckAuthenticatedAddresses = ({
     const promises = userActiveAccounts.map(async (activeAccount) => {
       const isAuth = await app.sessions
         .getSessionController(chainBase)
-        .hasAuthenticatedSession(canvasChainId, activeAccount.address);
+        ?.hasAuthenticatedSession(canvasChainId, activeAccount.address);
```

Comment:
do we need this?

Reply:
yeah, I encounter some console errors during work on create-community flow

Reply:
cc: @raykyri perhaps here?